config.go

```diff
@@ -405,6 +422,31 @@ func loadConfig() (*config, []string, error) {
 		}
 		fmt.Fprintln(os.Stderr, err)
 		return nil, nil, err
+
+	} else if cfg.CreateTemp == true {
+		tempWalletExists := false
+
+		// Error if the wallet already exists.
+		if fileExists(dbPath) {
+			err := fmt.Errorf("The wallet already exists. Loading this " +
+				"wallet instead.")
+			fmt.Fprintln(os.Stderr, err)
+			tempWalletExists = true
+		}
+
+		if !tempWalletExists {
+			// Ensure the data directory for the network exists.
+			if err := checkCreateDir(netDir); err != nil {
```

Comment:
Is this actually needed?  I believe you've added code to ensure this temp wallet is required to be somewhere besides the normal data directory.